src/compiler/scanner.ts

```diff
@@ -19,8 +19,12 @@ module ts {
         isReservedWord(): boolean;
         isUnterminated(): boolean;
         reScanGreaterToken(): SyntaxKind;
+        reScanLessThanSlashToken(): SyntaxKind;

```

Comment:
surprised this is needed.  Does this not happen implicitly when reScanJSXToken gets called?